poznaj/stories/tests/test_views.py

```diff
@@ -76,3 +79,45 @@ def test_update_story(self):
         self.assertEqual(database_story.title, 'new_title')
         self.assertEqual(database_story.description, 'new_description')
         self.assertEqual(str(database_story.duration), '1:00:00')
+
+    def test_filter_story(self):
```

Comment:
Nie wiem, jak do końca działa Pythonowy framework do testów, ale można by tu się pokusić o oddzielne testy:
* czy zwraca stories według odległości pierwszego punktu,
* czy sortuje wyniki według odległości (jeśli to jest istotne - ale widziałem, że jest tak zaimplementowane).

No i chyba asserty mogłyby być na samych id-kach - w sensie, że assertujemy, że np. story o id=1 zostało zwrócone, a niekoniecznie trzeba assertować, że description się zgadza.

Co o tym sądzisz?